app/src/main/java/org/wikipedia/edit/summaries/EditSummaryTag.kt

```diff
@@ -0,0 +1,56 @@
+package org.wikipedia.edit.summaries
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.LinearLayout
+import androidx.annotation.AttrRes
+import androidx.annotation.ColorInt
+import androidx.appcompat.widget.AppCompatTextView
+import androidx.core.content.res.ResourcesCompat
+import androidx.core.view.setMargins
+import androidx.core.view.setPadding
+import org.wikipedia.R
+import org.wikipedia.util.DimenUtil.dpToPx
+import org.wikipedia.util.ResourceUtil.getThemedAttributeId
```

Comment:
Please import `import org.wikipedia.util.DimenUtil` and `import org.wikipedia.util.ResourceUtil` here.